hidpi.sh

```diff
@@ -184,8 +184,8 @@ EEF
     get_edid
 
     thisDir=$(dirname $0)
-    thatDir="/System/Library/Displays/Contents/Resources/Overrides"
-    Overrides="\/System\/Library\/Displays\/Contents\/Resources\/Overrides"
+    thatDir="/Library/Displays/Contents/Resources/Overrides"
```

Comment:
A direct rename here will break so many things in the script. For example in https://github.com/xzhih/one-key-hidpi/blob/master/hidpi.sh#L191, the icon tiff files are located in the origin System directory, not Library directory.

Also, the script has to cover the old version user whose backup files are in the original System directory. The `disable()` function needs rewrite to support all users; otherwise, the disable function will only work for the new user.

There're lots of delicate stuffs in this script and let @xzhih have time to deal with them. No rush.

Reply:
ok